nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -625,6 +625,10 @@ sub generateNetworkingDhcpConfig {
 
         my $networkingDhcpConfig = generateNetworkingDhcpConfig();
 
+        my $desktopConfiguration = <<EOF;
```

Comment:
```suggestion
        (my $desktopConfiguration = <<EOF)=~s/^/  /gm;
```
This will add two spaces to in front of each line